spec/models/user_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Abel Morara', posts_counter: 5) }
+
+  before { subject.save }
+
+  it 'is valid with correct attributes' do
+    expect(subject).to be_valid
```

Comment:
- [ ] Let's kindly assert and attest that our tests are running successfully :+1: 
![Screenshot from 2023-10-13 03-37-41](https://github.com/Paulinelali/RoR-blog/assets/63087491/bc57a856-0f27-4f31-9827-48acadd1a12b)